packages/global-styles/src/components/global-styles-variations/index.tsx

```diff
@@ -4,7 +4,9 @@ import { useState } from '@wordpress/element';
 import { ENTER } from '@wordpress/keycodes';
 import classnames from 'classnames';
 import { translate, TranslateResult } from 'i18n-calypso';
-import { useMemo, useContext } from 'react';
+import { useMemo, useContext, useEffect } from 'react';
+// eslint-disable-next-line no-restricted-imports
```

Comment:
We shouldn't use Calypso imports in packages since they can be used out of Calypso. This is one of the reasons why we cannot avoid the property propagation.

Reply:
This is going to change once I use the final DS for the experiment, so I'll just ignore your comment.